packaging/linux/fedora/sunshine.spec

```diff
@@ -0,0 +1,237 @@
+%global build_timestamp %(date +"%Y%m%d")
+%undefine _hardened_build
+
+Name:           sunshine
+Version:        2024.911.215654  # TODO: can we get this dynamically?
+Summary:        Sunshine is a self-hosted game stream host for Moonlight.
+Release:        1%{?dist}
+License:        GPLv3-only
+URL:            https://github.com/LizardByte/Sunshine
+Source0:        sunshine.sysusers  # TODO: what is this?
```

Comment:
https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/#_dynamic_allocation this was my best attempt at following the fedora guidelines for adding a user to a group, i.e. for input.

Reply:
I think this is the last item...

Does this not take care of it though?

```bash
# Trigger udev rule reload for /dev/uinput and /dev/uhid
path_to_udevadm=$(which udevadm)
if [ -x "$path_to_udevadm" ] ; then
  $path_to_udevadm control --reload-rules
  $path_to_udevadm trigger --property-match=DEVNAME=/dev/uinput
  $path_to_udevadm trigger --property-match=DEVNAME=/dev/uhid
fi
```